markets/perps-market/subgraph/package.json

```diff
@@ -9,8 +9,8 @@
     "graph": "graph",
     "subgraphgen": "./codegen.sh",
     "auth": "graph auth --product hosted-service $V3_GRAPH_KEY",
-    "satsuma:optimism-goerli": "graph deploy perps-market-optimism-goerli subgraph.optimism-goerli.yaml --output-dir ./build/optimism-goerli --version-label v1 --node https://app.satsuma.xyz/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
-    "satsuma:base-goerli-competition": "graph deploy perps-market-base-testnet subgraph.base-goerli-competition.yaml --output-dir ./build/base-goerli-competition --version-label v3 --node https://app.satsuma.xyz/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
+    "alchemy:optimism-goerli": "graph deploy perps-market-optimism-goerli subgraph.optimism-goerli.yaml --output-dir ./build/optimism-goerli --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
```

Comment:
`https://ipfs.satsuma.xyz` endpoint is still correct. When it is migrated will do another update.